sql/moz-fx-data-shared-prod/org_mozilla_fenix/baseline_clients_daily/view.sql

```diff
@@ -1,8 +1,8 @@
 -- Generated via bigquery_etl.glean_usage
 CREATE OR REPLACE VIEW
-  `moz-fx-data-shared-prod.org_mozilla_fenix.baseline_clients_daily`
+  `org_mozilla_fenix.baseline_clients_daily`
```

Comment:
This violates our policy of having views include only fully-qualified references (which used to be required by BQ). I'm not sure if there's any good way around this while still enabling the tests here.

Reply:
I see, it makes sense in the context of publishing views to `mozdata` and the like too.

Reply:
I'm not sure if you meant the references in the view or the view itself e.g. `moz-fx-data-shared-prod.org_mozilla_fenix_derived.baseline_clients_daily_v1` vs `moz-fx-data-shared-prod.org_mozilla_fenix.baseline_clients_daily`. I'm assuming the former and that the latter can be implicit based on the default project of the bq client.

Reply:
In practice, we have used a fully-qualified name for the view we are creating as well, and we may have an implicit dependency on this in our view publishing script or in Jenkins logic (which replaces shared-prod with shared-nonprod for publishing to the nonprod project. We should make sure this is fully qualified here for consistency.